openhtf/plugs/__init__.py

```diff
@@ -417,7 +417,7 @@ def _create_or_update_rpc_server(self):
             attr_name not in plug.disable_remote_attrs):
           plug_methods.append((attr, '.'.join(('plugs', name, attr_name))))
 
-    if not plug_methods:
+    if not plug_methods or not conf.enable_station_api:
```

Comment:
There's already a conf option for doing this, it's setting a station_api_port of None, I think the only change you need is to check that here and leave the other stuff as-is.